internal_transfers/actions/tests/simulate-tenderly.spec.ts

```diff
@@ -0,0 +1,61 @@
+      "0",
+      false,
+      16300366
+    );
+    expect(simulation.transaction).toMatchSnapshot();
+  });
+
+  test.skip("simulate() returns expected output on failed simulation", async () => {
+    const simulation = await simulator.simulate(
+      SETTLEMENT_CONTRACT_ADDRESS,
+      oneAddress,
+      "0x",
+      "0",
+      false,
+      16300366
+    );
+    expect(simulation.transaction).toMatchSnapshot();
```

Comment:
The part i don't like of using snapshot, is that you are not asserting against some functional spec , you assert against a "picture of what it used to return at a given point in time"

Reply:
This tests was more to give rise to a concern I have about the fact that the simulation of a failing transaction is practically indistinguishable for a successful one. The only information I can tell that give any indication is that the gas used is quite low relative to most successful transactions. Thanks for flagging this concern regarding the snapshot, I will try to be more specific about the intentions of the test.